src/content/en/2020/accessibility.md

```diff
@@ -8,17 +8,442 @@ authors: [oluoluoxenfree, alextait1]
+
+We found that 15.50% of desktop pages and 14.62% of mobile pages contained at least one anchor element with `role=”button”`. If a role has been applied to an element that should have its implicit role respected, such as giving a `role=”button”` to a link (which has an implicit `role=”link”`), this would break the 2nd rule of ARIA and would violate [WCAG 2.1.1, Keyboard](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html) if the correct keyboard behavior has not been implemented (links are not activated with the space key, where as buttons are). 
+
+In the vast majority of these cases, a better pattern than explicitly defining `role=”button”` on the element in question would be to leverage the native HTML `<button>` element.
+
+##### Navigation
+
+We found that 22.06% of desktop pages and 21.76% of mobile pages have at least one element with `role=”navigation”`, which is a landmark role. Per the 1st rule of ARIA, rather than adding this role to an element, developers should be leveraging the HTML 5 `<nav>` element which comes with the correct semantics implicitly. It is possible that this role has been added explicitly to the `<nav>` element, which would not be an accessibility issue, though it is redundant.  
+
+##### Dialog Modals
+
+There are many potential accessibility barriers associated with dialog modals. We recommend reading Scott O’Hara’s article [Having an Open Dialog](https://www.scottohara.me/blog/2019/03/05/open-dialog.html) for more context. 
+
+We are pleased to report that 19.01% of desktop pages and 18.21% of mobile pages have at least one occurrence of `role=”dialog”` which is up from about 8% in 2019. It is worth noting some of the increase is probably due to changes in how this metric was measured.This could also suggest that more developers are considering accessibility when building dialogs and potentially that frameworks and associated packages may be implementing more accessible dialog patterns as well. However, making a dialog modal accessible requires a lot more than using the `dialog` role. Focus management, proper keyboard support, and screen reader communication all need to be addressed. 
+
+##### Tabs
+
+Tabs are a common interface widget, but present a challenge for many developers to make accessible. A common pattern for accessible implementation comes from  the [WAI-ARIA Authoring Practices Design Patterns](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel). Please note that the ARIA Authoring Practices document is not a specification, and is meant to show idealized ARIA constructs. They should not be used in production without testing with your users. 
+
+In this pattern, a parent container has a `role=”tablist”` with children elements that have a `role=”tab”`. These tabs are associated with elements that have a `role=”tabpanel”`, and contain the content for that tab. 
+
+{# TODO: ([Source](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html)) #}
```

Comment:
@bazzadp I got build errors trying to add this source link into the caption. Any idea how to get it to work?

Reply:
Can you show me the caption you're trying to use?

Reply:
```
caption="Tab list (`role=\"tablist\"`) contains all of the tabs. ([Source](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html))",
```

Reply:
You're including the quotes in code blocks so don't think you can also escape because it's repeated verbatim.

Just switch to single quotes and make life easier on yourself for this one. The following works:
```
  caption='Tab list (`role="tablist"`) contains all of the tabs. ([Source](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html))',
```

Reply:
Thanks!